fsspec/implementations/tests/test_local.py

```diff
@@ -508,16 +508,19 @@ def test_linked_files(tmpdir):
 
     fs = LocalFileSystem()
     assert fs.info(fn0)["type"] == "file"
-    assert fs.info(fn1)["type"] == "file"
-    assert fs.info(fn2)["type"] == "file"
+    assert fs.info(fn1)["type"] == "link"
+    assert fs.info(fn2)["type"] == "link"
 
     assert not fs.info(fn0)["islink"]
     assert fs.info(fn1)["islink"]
     assert fs.info(fn2)["islink"]
 
     assert fs.info(fn0)["size"] == len(data)
-    assert fs.info(fn1)["size"] == len(data)
-    assert fs.info(fn2)["size"] == len(data)
+
+    if not sys.platform == "win32":
+        # Posix symbolic link size is the lenght of the path it contains
+        assert fs.info(fn1)["size"] == len(fn0)
+        assert fs.info(fn2)["size"] == len(fn0)
```

Comment:
Previously we would get the size of the file it points to, which is what I would expect. And now we are getting the size of the link itself, which is platform-specific and I can't even think of a scenario where anyone would care about it.